bundle/manifests/sailoperator.clusterserviceversion.yaml

```diff
@@ -23,8 +23,18 @@ metadata:
     capabilities: Seamless Upgrades
     categories: OpenShift Optional, Integration & Delivery, Networking, Security
     containerImage: quay.io/maistra-dev/istio-operator:3.0-latest
-    createdAt: "2024-02-08T11:24:58Z"
+    createdAt: "2024-02-12T10:02:23Z"
     description: Experimental operator for installing Istio service mesh
+    features.operators.openshift.io/cnf: "false"
+    features.operators.openshift.io/cni: "true"
+    features.operators.openshift.io/csi: "false"
+    features.operators.openshift.io/disconnected: "true"
+    features.operators.openshift.io/fips-compliant: "true"
```

Comment:
This should probably be false, at least for now (Sail doesn't yet use our build of the proxy). Also note that the Sail operator is the equivalent of the community operator in 2.x. I believe we set the values for the product build outside of this repository. @FilipB should know more about this.

Reply:
Thanks Marko changes made there

Reply:
Better to check with OLM team, but I also think we should not include anything OCP specific to the community operator. On the other hand we will want to run community operator on both OCP and K8s so question for OLM team is how to do that.